src/commands/typescript/update.ts

```diff
@@ -6,13 +6,18 @@
  */
 
 import * as path from 'path';
+import * as glob from 'glob';
 import { flags, FlagsConfig, SfdxCommand } from '@salesforce/command';
 import { fs, Messages, SfdxError } from '@salesforce/core';
-import { exec } from 'shelljs';
+import { exec, pwd, cd } from 'shelljs';
 import { set } from '@salesforce/kit';
 import { AnyJson, asObject, getString } from '@salesforce/ts-types';
 import { NpmPackage, Package } from '../../package';
-import { SinglePackageRepo } from '../../repository';
+import { SinglePackageRepo, LernaRepo } from '../../repository';
+
+type LernaJson = {
```

Comment:
This is already defined here: https://github.com/salesforcecli/plugin-release-management/blob/main/src/repository.ts#L27